src/script/components/Modals/DetailViewModal/index.tsx

```diff
@@ -236,6 +233,16 @@ const DetailViewModal: FC<DetailViewModalProps> = ({
     }
   }, []);
 
+  useEffect(() => {
+    const onKeyDown = (event: KeyboardEvent): void => {
+      preventFocusOutside(event, 'detail-view');
```

Comment:
Maybe lets put this `'detail-view'` into a const, so if it ever changes, we're sure it will still work.